.env.example

```diff
@@ -1,7 +1,5 @@
-TEST_RPC_URL='' # rpc url to run tests with
-TEST_PRIVATE_KEY='0xSomething' # private key to run tests with
-TEST_CHAIN_ID=11155111 # chain id to run tests with
-TEST_ERC5564_DEPLOY_BLOCK=5281643 # deployment block number of the ERC5564 contract for tests
-TEST_LOCAL_NODE_ENDPOINT='http://127.0.0.1:8545' # endpoint of the local node: "listening on..."
-TEST_STEALTH_META_ADDRESS_URI='' # uri of the stealth meta address
-TEST_ENVIRONMENT_IS_LOCAL='true' # 'true', or 'false' if wanting to use remote (sepolia) chain; will be auto-set to false when running `bun run test-remote`
\ No newline at end of file
+# Fork testing environment variables
+TEST_RPC_URL='' # rpc url to run tests with if using a fork
+TEST_PRIVATE_KEY='0xSomething' # private key to run tests with if using a fork
+TEST_CHAIN_ID=11155111 # chain id to run tests with if using a fork
```

Comment:
Ok, so a few things:
1. We should just grab the chainid from the RPC URL when testing using a fork.
2. We should get rid of the `TEST` prefix for the env vars.
3. When I don't provide a private key and run `bun run test-fork` I'm getting `Details: insufficient funds for gas * price + value: balance 1, tx cost 1000000019000000, overshot 1000000018999999`
4. When I do provide a private key, it's actually sending transactions on sepolia instead of testing against the sepolia fork of the anvil node.

Reply:
for 1. should be handled by: https://github.com/ScopeLift/stealth-address-sdk/pull/37/commits/040496e7b14693823574185490c2d0776d87d349

Reply:
regarding 3: make sure to set `PRIVATE_KEY` in env
regarding 4: [ee554a2](https://github.com/ScopeLift/stealth-address-sdk/pull/37/commits/ee554a219556a2a74806b7bf5160a804f59a211f)

Reply:
Awesome, working well now. Thank you!